multinet/swagger/get_workspace_permissions.yaml

```diff
@@ -1,4 +1,4 @@
-Retrieve a single workspace.
+Retrieve the permissions on a workspace.
```

Comment:
Doesn't more in this file need to be updated?

Reply:
98a0c81